### PR TITLE
* Feat Added connection to Reports and File Management Microservices

### PR DESCRIPTION
**Pull request by JuanGuzman-io**

### CODE REVIEW

File: onecgiar-pr-server/src/api/platform-report/platform-report.service.ts
Comment (gcmartinezg) [inline onecgiar-pr-server/src/api/platform-report/platform-report.service.ts:1]: @JuanGuzman-io can we change the name of the env variables to be something more meaningful? CLA_MICROSERVICE2 and CLA_MICROSERVICE4 are not too clear, in my opinion.

**Review by gcmartinezg — COMMENTED** [review]
Please check the comment I left.

File: onecgiar-pr-server/src/api/platform-report/platform-report.service.ts
Comment (JuanGuzman-io) [reply onecgiar-pr-server/src/api/platform-report/platform-report.service.ts:1]: Maybe we can use the Microservice name directly? Like REPORTS_USER and FILE_MANAGEMENT_USER.

**Review by gcmartinezg — APPROVED** [review]
Looks good

**Pull request merged**